packages/fluentui/docs/src/examples/components/Popup/Types/PopupExamplePointing.shorthand.tsx

```diff
@@ -101,7 +101,13 @@ const triggers = [
 ];
 
 const PopupExamplePointing = () => (
-  <Grid columns="repeat(3, 30px)" variables={{ padding: '30px', gridGap: '80px' }}>
+  <Grid
+    styles={{
+      gridTemplateColumns: 'repeat(3, 30px)',
+      msGridColumns: '(30px)[3]',
+    }}
+    variables={{ padding: '30px', gridGap: '80px' }}
```

Comment:
We have `gridGap` there, but it will not work in IE

Reply:
Alright, update to `'30px 80px 30px 80px 30px'`